api/v1/installation_types.go

```diff
@@ -764,3 +790,25 @@ func IsFIPSModeEnabled(mode *FIPSMode) bool {
 func IsFIPSModeEnabledString(mode *FIPSMode) string {
 	return fmt.Sprintf("%t", IsFIPSModeEnabled(mode))
 }
+
+type WindowsNodeSpec struct {
+	// CNIBinDir is the path to the CNI binaries directory on Windows, it must match what is on ContainerD on the Windows nodes.
+	// +optional
+	CNIBinDir string `json:"cniBinDir,omitempty"`
+
+	// CNIConfDir is the path to the CNI configuration directory on Windows, it must match what is on ContainerD on the Windows nodes.
+	// +optional
+	CNIConfDir string `json:"cniConfDir,omitempty"`
+
+	// CNILogDir is the path to the Calico CNI logs directory on Windows.
+	// +optional
+	CNILogDir string `json:"cniLogDir,omitempty"`
```

Comment:
I'm not entirely sure on the inner workings, but it seems to me as the configs in CNILogging should be the same on Windows (log level, file max sizes, etc). Does this require any extra work on cni-plugin in the calico repo? Otherwise I think I should add these to the windows rendering...

Reply:
I'm really not sure what our Windows CNI plugin supports in terms of logging, so you might need to dig into that one a bit :sweat_smile: